test/test_enclave_wrangler.py

```diff
@@ -133,6 +148,21 @@ def test_upload_cset_container_from_csv(self):
         response: Dict = upload_new_cset_container_with_concepts_from_csv(df=df)
         print()
 
+    def test_codeset_version_enclave_to_db(self):
```

Comment:
@Sigfried FYI: Updated test function. This tests that inserting something that already exists fails, and inserting something that doesn't exist in the table correctly adds.